packages/verifiable-credentials/src/credentials.ts

```diff
@@ -128,29 +144,50 @@ export default class Credentials {
 	 * @param data 
 	 * @returns 
 	 */
-	async createCredentialJWT(data: any): Promise<object> {
+	async createCredentialJWT(subjectId: string, data: any, expirationDate?: string): Promise<object> {
+		// Ensure a credential schema has been specified
+		if (!data.schema) {
+			throw new Error('No schema specified')
+		}
+
+		// Ensure data matches specified schema
+		const schema = await this.context.getClient().getSchema(data.schema)
+		const isValid = await schema.validate(data);
+
+		// @todo: Check the schema is a "credential" schema type?
```

Comment:
@cmcWebCode40 Can you add an issue for this?

Reply:
Backlog issue for this task her 
 https://github.com/verida/verida-js/issues/146